docs/system-admin-guide/design/README.md

```diff
@@ -11,46 +11,51 @@ As an OpenProject Enterprise add-on you can replace the default  OpenProject log
 
 Navigate to *Administration* -> *Design* in order to customize your OpenProject theme and logo.
 
-The design page provides several options to customize your OpenProject Enterprise edition:
+The design page provides several options to customize your OpenProject Enterprise edition, grouped under three tabs, **Interface, Branding, PDF export styles**. You can [choose a color theme](#choose-a-color-theme) under any of these tabs. 
 
-1. Choose a default color theme: OpenProject, Light or Dark. Press the Save button to apply your changes.
-2. Upload your own **custom logo** to replace the default OpenProject logo.
-3. Set a custom **favicon** which is shown as an icon in your browser window/tab.
-4. Upload a custom **touch icon** which is shown on your smartphone or tablet when you bookmark OpenProject on your home screen.
-5. Set the **Custom PDF export settings** for [exporting work packages in a PDF format](../../user-guide/work-packages/exporting/#pdf-export).
-6. [Advanced settings](#advanced-settings) to configure **custom colors** to adjust nearly any aspect of OpenProject, such  as the color of the header and side menu, the link color and the hover color.
+Under **Interface** you can also set [the interface colors](#interface-colors) to configure **custom colors** to adjust nearly any aspect of OpenProject, such  as the color of the header and side menu, the link color and the hover color.
```

Comment:
I would change this a little since the goal is to reduce the number of options for the user; in the future, we'll go towards asking the user for just one or two colours and automatically generate a theme based on those. Suggestion: 

> Under **Interface**, you can also choose [custom colors](#interface-colors) for elements of the interface such as the primary link colour, secondary accent colour, the background of the top navigation header and the main menu.